src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/ExceptionsPipelineTests.cs

```diff
@@ -273,7 +273,7 @@ public Task EventExceptionsPipeline_InnerUnthrownException()
 
                     IExceptionInstance outerInstance = instances.Skip(1).Single();
                     Assert.NotNull(outerInstance);
-                    
+
```

Comment:
I don't believe there's currently anything that expects failures for <8, but IIRC Justin added in logic to use `ApplyStartupHook` instead of setting the environment variable for .NET 8+ in [this PR](https://github.com/dotnet/dotnet-monitor/pull/4746/files), which lets us exercise the new method as well as the existing way of setting it. If people want additional testing for this PR, I can add that though.

Reply:
@jander-msft I know we chatted offline about this the other day - did you have any thoughts about if we want to add functional tests for Exceptions in this PR?

Reply:
I'm fine with a follow up PR to do this. There currently aren't any functional tests for this feature, so it would be a bit of an undertaking.

Reply:
/todo Add Functional Tests For Exceptions Feature